### ISSUE TITLE
Can't compile

### ISSUE THREAD

**Issue by franksouza183** [dragonfireclient/dragonfireclient#41]
I'm trying  to  compile on manjaro and that's what I got:
```
$ cmake . -DRUN_IN_PLACE=TRUE -DIRRLICHT_LIBRARY=$PWD/irrlicht/lib/Linux/libIrrlichtMt.so -DIRRLICHT_INCLUDE_DIR=$PWD/irrlicht/include
-- The C compiler identification is GNU 11.1.0
-- The CXX compiler identification is GNU 11.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- *** Will build version 5.5.0-dragonfire ***
-- Found Irrlicht: /home/frank/Downloads/df/dragonfire/dragonfireclient/irrlicht/lib/Linux/libIrrlichtMt.so  
-- Looking for IRRLICHT_VERSION_MT
-- Looking for IRRLICHT_VERSION_MT - found
-- Using GMP provided by system.
-- Found GMP: /usr/lib/libgmp.so  
-- Using JsonCpp provided by system.
-- Found Json: /usr/lib/libjsoncpp.so  
-- Found LuaJIT: /usr/lib/libluajit-5.1.so (found version "") 
-- Using LuaJIT provided by system.
-- Found CURL: /usr/lib/libcurl.so  
-- cURL support enabled.
-- Found GettextLib: /usr/include  
-- GetText enabled; locales found: ar;be;bg;ca;cs;da;de;dv;el;eo;es;et;eu;fi;fr;gd;gl;he;hi;hu;id;it;ja;jbo;kk;kn;ko;ky;lt;lv;ms;ms_Arab;nb;nl;nn;pl;pt;pt_BR;ro;ru;sk;sl;sr_Cyrl;sr_Latn;sv;sw;th;tr;uk;vi;zh_CN;zh_TW
-- Found OpenAL: /usr/lib/libopenal.so  
-- Found Vorbis: /usr/include  
-- Sound enabled.
-- Found OpenGL: /usr/lib/libOpenGL.so   
-- Found Freetype: /usr/lib/libfreetype.so (found version "2.10.4") 
-- Freetype enabled.
-- Looking for cbreak in /usr/lib/libncursesw.so
-- Looking for cbreak in /usr/lib/libncursesw.so - found
-- Found Ncursesw: /usr/lib/libncursesw.so  
-- ncurses console enabled.
-- Found PostgreSQL: /usr/lib/libpq.so (found version "13.3")  
-- PostgreSQL backend enabled
-- PostgreSQL includes: /usr/include
-- LevelDB backend enabled.
-- Redis not found!
-- Found SQLite3: /usr/lib/libsqlite3.so  
-- Prometheus client disabled.
-- SpatialIndex AreaStore backend enabled.
-- Found X11: /usr/include   
-- Looking for XOpenDisplay in /usr/lib/libX11.so;/usr/lib/libXext.so
-- Looking for XOpenDisplay in /usr/lib/libX11.so;/usr/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found JPEG: /usr/lib/libjpeg.so (found version "80") 
-- Found ZLIB: /usr/lib/libz.so (found version "1.2.11") 
-- Found PNG: /usr/lib/libpng.so (found version "1.6.37") 
-- Looking for clock_gettime in rt
-- Looking for clock_gettime in rt - found
-- Looking for include file endian.h
-- Looking for include file endian.h - found
-- Locale blacklist applied; Locales used: be;bg;ca;cs;da;de;el;eo;es;et;eu;fi;fr;gd;gl;hu;id;it;ja;jbo;kk;ko;lt;lv;ms;nb;nl;nn;pl;pt;pt_BR;ro;ru;sk;sl;sr_Cyrl;sr_Latn;sv;sw;tr;uk;vi;zh_CN;zh_TW
-- Performing Test IS_AARCH64
-- Performing Test IS_AARCH64 - Failed
-- Found Doxygen: /usr/bin/doxygen (found version "1.9.1") found components: doxygen dot 
-- Configuring done
-- Generating done
-- Build files have been written to: /home/frank/Downloads/df/dragonfire/dragonfireclient
```

--- Trying to compile ...

```
$ make -j$(nproc)
[  0%] mo-update [el]: Creating locale directory.
[  0%] mo-update [da]: Creating locale directory.
[  0%] mo-update [bg]: Creating locale directory.
[  0%] mo-update [be]: Creating locale directory.
[  0%] mo-update [cs]: Creating locale directory.
[  0%] mo-update [eu]: Creating locale directory.
[  1%] mo-update [de]: Creating locale directory.
[  3%] mo-update [ca]: Creating locale directory.
[  3%] mo-update [et]: Creating locale directory.
[  3%] mo-update [eo]: Creating locale directory.
[  3%] mo-update [es]: Creating locale directory.
[  3%] Built target GenerateVersion
[  3%] mo-update [fr]: Creating locale directory.
[  4%] mo-update [gd]: Creating locale directory.
[  4%] mo-update [fi]: Creating locale directory.
[  5%] mo-update [it]: Creating locale directory.
[  5%] mo-update [hu]: Creating locale directory.
[  5%] mo-update [gl]: Creating locale directory.
[  5%] mo-update [id]: Creating locale directory.
[  5%] mo-update [jbo]: Creating locale directory.
[  5%] mo-update [kk]: Creating locale directory.
[  5%] mo-update [ja]: Creating locale directory.
[  6%] mo-update [ko]: Creating locale directory.
[  6%] mo-update [lt]: Creating locale directory.
[  6%] mo-update [lv]: Creating locale directory.
[  7%] mo-update [ms]: Creating locale directory.
[  7%] mo-update [nb]: Creating locale directory.
[  7%] mo-update [nl]: Creating locale directory.
[  7%] Building CXX object src/CMakeFiles/minetest.dir/client/activeobjectmgr.cpp.o
[  7%] mo-update [nn]: Creating locale directory.
[  8%] mo-update [pl]: Creating locale directory.
[  8%] Building CXX object src/CMakeFiles/minetest.dir/client/camera.cpp.o
[  8%] mo-update [pt]: Creating locale directory.
[  9%] Building CXX object src/CMakeFiles/minetest.dir/client/client.cpp.o
[  9%] Building CXX object src/CMakeFiles/minetest.dir/client/clientenvironment.cpp.o
[  9%] Building CXX object src/CMakeFiles/minetest.dir/client/clientlauncher.cpp.o
[  9%] Building CXX object src/CMakeFiles/minetest.dir/client/clientmap.cpp.o
[  9%] mo-update [ro]: Creating locale directory.
[ 10%] mo-update [sk]: Creating locale directory.
[ 10%] mo-update [ru]: Creating locale directory.
[ 10%] mo-update [pt_BR]: Creating locale directory.
[ 11%] Building CXX object src/CMakeFiles/minetest.dir/client/clientmedia.cpp.o
[ 11%] Building CXX object src/CMakeFiles/minetest.dir/client/clientobject.cpp.o
[ 11%] mo-update [sl]: Creating locale directory.
[ 12%] mo-update [sr_Latn]: Creating locale directory.
[ 12%] mo-update [sr_Cyrl]: Creating locale directory.
[ 12%] Building CXX object src/CMakeFiles/minetest.dir/client/clouds.cpp.o
[ 12%] mo-update [sv]: Creating locale directory.
[ 12%] Building CXX object src/CMakeFiles/minetest.dir/client/content_cao.cpp.o
[ 12%] mo-update [sw]: Creating locale directory.
[ 12%] mo-update [tr]: Creating locale directory.
[ 13%] mo-update [uk]: Creating locale directory.
[ 14%] Building CXX object src/CMakeFiles/minetest.dir/client/content_cso.cpp.o
[ 14%] mo-update [vi]: Creating locale directory.
[ 14%] mo-update [zh_CN]: Creating locale directory.
[ 15%] mo-update [zh_TW]: Creating locale directory.
[ 16%] mo-update [be]: Creating mo file.
/home/frank/Downloads/df/dragonfire/dragonfireclient/src/client/content_cao.cpp:23:10: fatal error: ITextSceneNode.h: Arquivo ou diretório inexistente
   23 | #include <ITextSceneNode.h>
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [src/CMakeFiles/minetest.dir/build.make:202: src/CMakeFiles/minetest.dir/client/content_cao.cpp.o] Erro 1
make[2]: ** Esperando que outros processos terminem.
[ 16%] mo-update [bg]: Creating mo file.
[ 16%] mo-update [ca]: Creating mo file.
[ 16%] mo-update [cs]: Creating mo file.
[ 17%] mo-update [da]: Creating mo file.
[ 17%] mo-update [de]: Creating mo file.
[ 17%] mo-update [el]: Creating mo file.
[ 18%] mo-update [eo]: Creating mo file.
[ 18%] mo-update [es]: Creating mo file.
[ 18%] mo-update [et]: Creating mo file.
[ 18%] mo-update [eu]: Creating mo file.
[ 19%] mo-update [fi]: Creating mo file.
[ 19%] mo-update [fr]: Creating mo file.
[ 19%] mo-update [gd]: Creating mo file.
[ 19%] mo-update [gl]: Creating mo file.
[ 20%] mo-update [hu]: Creating mo file.
[ 20%] mo-update [id]: Creating mo file.
[ 20%] mo-update [it]: Creating mo file.
[ 20%] mo-update [ja]: Creating mo file.
[ 21%] mo-update [jbo]: Creating mo file.
[ 21%] mo-update [kk]: Creating mo file.
[ 21%] mo-update [ko]: Creating mo file.
[ 21%] mo-update [lt]: Creating mo file.
[ 22%] mo-update [lv]: Creating mo file.
[ 22%] mo-update [ms]: Creating mo file.
[ 22%] mo-update [nb]: Creating mo file.
[ 22%] mo-update [nl]: Creating mo file.
[ 23%] mo-update [nn]: Creating mo file.
[ 23%] mo-update [pl]: Creating mo file.
[ 23%] mo-update [pt]: Creating mo file.
[ 24%] mo-update [pt_BR]: Creating mo file.
[ 24%] mo-update [ro]: Creating mo file.
[ 24%] mo-update [ru]: Creating mo file.
[ 24%] mo-update [sk]: Creating mo file.
[ 25%] mo-update [sl]: Creating mo file.
[ 25%] mo-update [sr_Cyrl]: Creating mo file.
[ 25%] mo-update [sr_Latn]: Creating mo file.
[ 25%] mo-update [sv]: Creating mo file.
[ 26%] mo-update [sw]: Creating mo file.
[ 26%] mo-update [tr]: Creating mo file.
[ 26%] mo-update [uk]: Creating mo file.
[ 26%] mo-update [vi]: Creating mo file.
[ 27%] mo-update [zh_CN]: Creating mo file.
[ 27%] mo-update [zh_TW]: Creating mo file.
[ 27%] mo update
[ 27%] Built target translations
make[1]: *** [CMakeFiles/Makefile2:420: src/CMakeFiles/minetest.dir/all] Erro 2
make: *** [Makefile:156: all] Erro 2
```

**Comment by LizzyFleckenstein03** [comment]
please update and try again, should have been fixed in f122888148f6efb2f4d28d4de4e4f4221cb29ac0

**Comment by franksouza183** [comment]
That works!
